bson/build.gradle

```diff
@@ -27,6 +27,15 @@ clirr {
     failOnErrors = true
 }
 
+sourceSets {
```

Comment:
This may be a mistake to only do this srcDir split for bson, and not driver-core.  There are a lot more classes moving into bson, which made it more compelling there.

Reply:
Interesting, we can always split out in the future.